simulator/lang/lib/index.ts

```diff
@@ -22,15 +22,17 @@ const filter = (obj: any, predicate: any) =>
 
 
 class ReachValue {
```

Comment:
I think in this file you should use the same names as the CBR... `T_Null` and `T_UInt` etc

Reply:
Or `R${nameInReach}` like `RUInt` and `RTuple`